packages/firebase_admob/android/src/main/java/io/flutter/plugins/firebaseadmob/FirebaseAdMobPlugin.java

```diff
@@ -154,6 +154,15 @@ private void callShowAd(int id, MethodCall call, Result result) {
     result.success(Boolean.TRUE);
   }
 
+  private void callIslAdLoaded(int id, MethodCall call, Result result) {
```

Comment:
I assume this method should be named `callIsAdLoaded` instead of `callIslAdLoaded`?

Reply:
Updated method name.